app/javascript/src/components/editor/TranslationKeys/TranslationKeysEditStrings.svelte

```diff
@@ -32,9 +33,9 @@
     $translationKeys = { ...$translationKeys }
 
     dispatch("updateKey", value)
-  }
+  }, 300)
 
-  function removeKey() {
+  const removeKey = () => {
```

Comment:
Regular functions are fine. Why did you choose to refactor this to be an arrow function?

Reply:
I was forced to by `renameKey`, since I wrapped it inside a `debounce`. Admittedly, I could've left that as an anonymous function as well, but a bit of personal preference swooped in as well.

I checked on a random file that we also sometimes use arrow functions at the root level of Svelte component scripts, just to be sure you'd maybe be okay with it 😅.

Since `renameKey` was changed, it didn't seem nice to have `removeKey` be a regular function too, so I changed that as well.

In short: personal preference + a change that somewhat justifies it.